kong/pdk/private/phases.lua

```diff
@@ -14,6 +14,7 @@ local PHASES = {
   rewrite           = 0x00000010,
   access            = 0x00000020,
   balancer          = 0x00000040,
+  response          = 0x00000080,
```

Comment:
Just a thought, have we considered making the phase name reflect more on what it does? For example, what about `buffered_response`. That way a new Kong developer won't get confused about why this fake phase exists and be always aware of the performance penalties associated with using it.

Reply:
that was my first proposal, but after quite a bit of bikeshedding we settled down on just `response`.